src/community/smart-data-loader/src/main/java/org/geoserver/smartdataloader/data/SmartDataLoaderDataAccessFactory.java

```diff
@@ -10,13 +10,7 @@
 import java.net.URI;
 import java.net.URL;
 import java.sql.SQLException;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
I'm pretty sure wildcard imports are not allowed, but somehow the QA / PMD workflow has not picked up on this... 

see also: https://github.com/geoserver/geoserver/blob/6c2ffdf81676c624c3216b680df0963a1968bcd8/build/qa/pmd-ruleset.xml#L105-L117

Reply:
Hi @mprins thanks for your feedback, fixing it.